File: Lab_1.php
<?php
$n = 0;
$i = 0;
$s = 0;
echo ("Please enter value for N\n");
$n = (int)readline();
$s = 0;
for ($i = 1; $i < $n; $i++) {
    if ($i % 2 == 0) {
        $s = $s + $i;
        echo("Current value i=" . $i . " S=" . $s);
        echo("\n");
    }
}
echo("Final result sum is S=" . $s);
?>

File: Lab_1.cs
using System;

class Program
{
    static void Main()
    {
        int n, i, s;
        s = 0;
        Console.WriteLine("Please enter value for N");
        n = Convert.ToInt32(Console.ReadLine());
        for (i = 1; i < n; i++)
        {
            if (i % 2 == 0)
            {
                s = s + i;
                Console.WriteLine("Current value i=" + i + " S=" + s);
            }
        }
        Console.WriteLine("Final result sum is S=" + s);
    }
}
